.github/workflows/ci.yml

```diff
@@ -51,6 +51,11 @@ jobs:
           sleep 30
           kubectl -n ingress-nginx get all
 
+      - name: Create secret
```

Comment:
The test suite uses the values files in `test/cases/`. Instead of hardcoding a license there, we can create the "bindplane" secret with only the `license` key. I added a license to the repo's secrets.